CONTRIBUTING.md

```diff
@@ -120,3 +121,26 @@ go get github.com/debber/debber-v0.3/cmd/...
 bin/server_package_debian 0.0.6 amd64
 bin/client_package_debian 0.0.6 amd64
 ```
+
+##Creating pull request
+
+To contribute a code, fist You must create a pull request (PR). If Your changes will be accepted
+this PR will be merged into main branch.
+
+Before creating PR be sure to: 
+
+* **Step 1.** Check if all files were formatted with `go fmt`
+
+```bash
+bin/check_go_fmt
+```
+
+* **Step 2.** Check if all unit tests passes; no vet and linter errors remain.
+
+```bash
+bin/test_commit
```

Comment:
`test_commit` runs linter on all touched files. This can result in unrelated errors in touched files - we shouldn't force contributors to fix unrelated linter problems.